### PR TITLE
add timestamps to out and web

### PR DESCRIPTION
**Pull request by DavidKorczynski**
This is useful for tracking performance.

### CODE REVIEW

**Comment by DavidKorczynski** [comment]
/gcbrun exp -n dk-test-infra-99x2 -m vertex_ai_gemini-1-5 -b minor-for-ci -i

**Comment by DavidKorczynski** [comment]
These are the visuals added:

Screenshot 2024-07-16 093954

https://llm-exp.oss-fuzz.com/Result-reports/ofg-pr/2024-07-16-485-dk-test-infra-99x2-minor-for-ci/index.html

File: run_all_experiments.py [hunk 1]
```diff
@@ -305,10 +310,33 @@ def _print_experiment_results(results: list[Result]):
           f'\n{result.result}\n')
 
 
+def add_to_json_report(outdir: str, key: str, value: Any) -> None:
+  """Adds a key/value pair to JSON report."""
+  if not os.path.isdir(outdir):
+    os.mkdir(outdir)
```
Comment (DonggeLiu) [inline run_all_experiments.py:316]: nit: `os.makedirs(outdir, exist_ok=True)`, no need to check.

**Review by DonggeLiu — APPROVED** [review]

**Comment by DavidKorczynski** [comment]
/gcbrun exp -n dk-test-infra-99x23 -m vertex_ai_gemini-1-5 -b minor-for-ci -i

**Comment by DavidKorczynski** [comment]
Experiment is looking good

**Pull request merged**